apps/developer-hub/src/components/EntropyTable/index.tsx

```diff
@@ -0,0 +1,125 @@
+"use client";
```

Comment:
Not something you need to worry about @aditya520 but just a note to self, we should probably look into using async server components from markdown to see if it's possible cc @alexcambose